src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -351,3 +377,15 @@ def __update_ui(self, robot_status: RobotStatus) -> None:
         )
 
         self.battery_progress_bar.setValue(power_status.battery_voltage)
+
+    def __calculate_average_round_trip_time(self, new_time: float) -> int:
+        """
+        Logs the given round-trip time and calculates the average
+        :param new_time: The new round-trip time to add
+        :return: The mean integer value of the previous round-trip times in milliseconds
+        """
+        self.previous_primitive_rtt_values.append(new_time)
+        average_rtt_time_milliseconds = sum(self.previous_primitive_rtt_values)
```

Comment:
```suggestion
        sum_rtt_time_milliseconds = sum(self.previous_primitive_rtt_values)
```

would probably be a better name